package.json

```diff
@@ -20,6 +20,6 @@
   "devDependencies": {
     "chai": "^4.2.0",
     "eslint": "^6.8.0",
-    "mocha": "^7.0.1"
+    "mocha": "^7.1.0"
```

Comment:
Based on this commit `added mocha/chai` I suspect you are running `npm i -D mocha chai`. Remember, when there is already a `package.json` we can just run `npm i` and it will install all of the listed dependencies.